dts/arm/nxp/nxp_rt.dtsi

```diff
@@ -357,6 +357,28 @@
 			status = "disabled";
 			label = "USBD_2";
 		};
+
+		usdhc1: usdhc@402c0000 {
+			compatible = "nxp,imx-usdhc";
+			reg = <0x402c0000 0x4000>;
+			status = "disabled";
+			interrupts = <110 0>;
+			clocks = <&ccm IMX_CCM_USDHC1_CLK 0 0>;
+			label = "USDHC_1";
+			#address-cells = <1>;
+			#size-cells = <0>;
```

Comment:
Properties `#address-cells` and `#size-cells` should be removed as usdhc nodes don't have child nodes (subnodes).

Reply:
fixed.

Reply:
You forgot to handle `#address-cells` and `#address-cells` in `mmc.yaml`. It should be optional, or better, we could remove them as they aren't used (we could add them later if a new node needs them).

Reply:
I removed them from mmc.yaml